### ISSUE TITLE
MCPE 0.14 ID mismatch (Schematic Import)

### ISSUE THREAD

**Issue by ShinkoNet** [Khroki/MCEdit-Unified#721]
When importing a schematic to a MCPE world, the new 0.14 blocks come up as wool.
Before Import:
https://puu.sh/r3Wg9/26860bab54.png
After Import:
https://puu.sh/r3Wdy/ede77e8f19.png

When loading the world afterwards even after importing the .schematic, everything is as expected.

**Comment by ShinkoNet** [comment]
nice tag, I can see that this thing is really quite the bug.

**Comment by naor2013** [comment]
Can you give us more info? Only 0.14 blocks become wool or does it happen to all the blocks? When opening the map in game, does it still shows them as wool or does it show them as the blocks they're supposed to be?

**Comment by LaChal** [comment]
As far as I can see on the screen-shot, only some blocks a changed to wool.
Did you created the schematic by selecting something in a PE world, or in a PC world?

**Comment by ShinkoNet** [comment]
Only the 0.14 blocks become wool. The rest of the blocks import fine.

The schematic was exported from the program Minecraft Note Block Studio, but it shouldn't matter much since the .schematic format is the same on each version.

**Comment by LaChal** [comment]
Looks like all redstone dust and repeater are converted to whool.
Did you try to import the schematic in a PC world?

**Comment by ShinkoNet** [comment]
Yes, and it works fine like always (This schematic has been working fine in PC version since so 2013)

**Comment by LaChal** [comment]
I guess I found why you have this bug.
I hope we can fix it for the next release :smile:

**Comment by ShinkoNet** [comment]
Been two releases, its improved slightly?

Repeaters are now pasted correctly, but buttons, redstone dust and note blocks still turn to wool.
![mcedit_2017-01-25_12-50-32](https://cloud.githubusercontent.com/assets/10773386/22274871/0dcdbe5c-e2fd-11e6-854d-b73d02035dab.png)

**Comment by Khroki** [comment]
I just noticed this report, sorry. It does look like the pocket.json is completely missing redstone dust, buttons, and note blocks in it's definitions. Which would be the cause of this. Looks like when it was converted from Yaml some entries failed to get transferred. I'll upload the new json here once it's fixed. Alternatively if you don't want to wait you can add them yourself to the Pocket.json file, the information needed is in the Pocket.yaml file. Both open with regular text editors.

**Comment by josevazqm** [comment]
Any updates on this?

**Comment by Podshot** [comment]
We're doing our best to support PE and update our block definitions right now. We'll have a better update on PE block definitions when we get closer to release.
